airbyte-metrics/reporter/src/main/java/io/airbyte/metrics/reporter/MetricRepository.java

```diff
@@ -140,6 +140,30 @@ having count(*) < 1440 / cast(c.schedule::jsonb->'units' as integer)
         + ctx.fetchOne(queryForAbnormalSyncInMinutesInLastDay).get("cnt", long.class);
   }
 
+  long numberOfJobsRunningUnusuallyLong() {
+    // Definition of unusually long means runtime is more than 2x historic avg run time or 15
+    // minutes more than avg run time, whichever is greater.
+    final var query =
+        """
+        select
+        	scope as connection_id,
+        	avg(extract(epoch from age(updated_at, created_at))) as avg_run_sec,
+        	extract(epoch from age(max(updated_at), max(created_at))) as latest_run_sec
+        from
+        	jobs
+        where
+        	updated_at >= NOW() - interval '168 HOUR'
+        	and jobs.status = 'succeeded'
+        	and jobs.config_type = 'sync'
+        group by 1
+        having
+        	count(1) > 4
+        	and extract(epoch from age(max(updated_at), max(created_at))) > greatest(avg(extract(epoch from age(updated_at, created_at))) * 2, avg(extract(epoch from age(updated_at, created_at))) + 900)
```

Comment:
can we add a comment explaining what this line is doing? what does 900 stand for?